### ISSUE TITLE
Wrong "Unused Opens" detected

### ISSUE THREAD

**Issue by SCullman** [ionide/ionide-vscode-fsharp#598]
Detected-> Types is not used
![image](https://user-images.githubusercontent.com/5693190/31759301-434f488a-b4b1-11e7-924c-647a12d67c98.png)
Well, removing "Types" is not a good idea here: 
![image](https://user-images.githubusercontent.com/5693190/31759396-8c66b012-b4b1-11e7-977e-506009e7fe80.png)
Tested with Ionide 3.8.0

**Comment by cmeeren** [comment]
Was just about to report this myself.

**Comment by cmeeren** [comment]
Dead simple repro:

![image](https://user-images.githubusercontent.com/7766733/31768829-53027016-b4d1-11e7-9b13-0a65d4567742.png)

![image](https://user-images.githubusercontent.com/7766733/31768849-644e49da-b4d1-11e7-9a1c-6969b5ec353a.png)

**Comment by SCullman** [comment]
Most times "Unused Opens" are detected correctly.
In my case the project has several modules:

- Types
- TopicA.Types
- TopicB.Types

The example above starts
```
module TopicA.State
open Types // which points to TopicA.Types
```

**Comment by vasily-kirichenko** [comment]
Relative opens are not supported. Use fully qualified.

**Comment by vasily-kirichenko** [comment]
![1](https://user-images.githubusercontent.com/873919/31771615-72458602-b4e5-11e7-8503-687337540399.gif)

**Comment by forki** [comment]
@vasily-kirichenko try 

    module Foo =
       ...

**Comment by cmeeren** [comment]
@vasily-kirichenko that looks like VS? How is that relevant?

**Comment by vasily-kirichenko** [comment]
![image](https://user-images.githubusercontent.com/873919/31771732-d9c14096-b4e5-11e7-9494-8bb61aa46250.png)

**Comment by forki** [comment]
@cmeeren it's relevant because we share the code ;-)

**Comment by vasily-kirichenko** [comment]
(it's on VFT master)

**Comment by Krzysztof-Cieslak** [comment]
@cmeeren, Ionide (or rather FSAC) implementation is based on VF# implementation. So @vasily-kirichenko is testing if the problem is in original implementation or I've introduce bug during porting.

**Comment by forki** [comment]
@vasily-kirichenko it's weird because I definetly saw something similar. Let me find the sample

**Comment by cmeeren** [comment]
Oh, OK. Then it might be fixed in some next version as per https://github.com/Microsoft/visualfsharp/issues/3347.

**Comment by vasily-kirichenko** [comment]
@forki I've fixed a bug in it yesterday, but it's not related to this one, see https://github.com/Microsoft/visualfsharp/pull/3776

**Comment by vasily-kirichenko** [comment]
@cmeeren I doubt https://github.com/Microsoft/visualfsharp/issues/3347 can be fixed without full rewrite of the feature.

**Comment by vasily-kirichenko** [comment]
@forki It's likely fixed in https://github.com/Microsoft/visualfsharp/pull/3776

**Comment by vasily-kirichenko** [comment]
@forki No, it's not. Minimal repro:

![image](https://user-images.githubusercontent.com/873919/31773573-d9e27cce-b4eb-11e7-977e-7604393cad0f.png)

**Comment by vasily-kirichenko** [comment]
Fixed in https://github.com/Microsoft/visualfsharp/pull/3776, please try

![image](https://user-images.githubusercontent.com/873919/31775492-6fd3e70e-b4f1-11e7-832a-63d1f60d3b59.png)

**Comment by Krzysztof-Cieslak** [comment]
Latest fix by @vasily-kirichenko is included in `3.8.1`
